src/ui/server/helpers/mappings/map-name-fields/index.test.ts

```diff
@@ -28,6 +31,13 @@ const mockApplicationWithCharacterCodes = {
     ...mockApplication.company,
     [COMPANY_NAME]: mockStringWithCharacterCodes,
   },
+  exportContract: {
+    ...mockApplication.exportContract,
+    agent: {
+      ...mockApplication.exportContract.agent,
+      [AGENT_NAME]: `${mockApplication.exportContract.agent}'`,
```

Comment:
Rather than a ' at the end, could this be changed to be like O'something so more realistic?

Reply:
Good shout, i've updated all agent name test mocks to contain `O'Neill`